packages/web/src/components/user-list-modal/components/lists/FollowersUserList.tsx

```diff
@@ -0,0 +1,35 @@
+import { useFollowers } from '@audius/common/api'
+import { followersUserListSelectors } from '@audius/common/store'
+import { useSelector } from 'react-redux'
+
+import { UserListV2 } from 'components/user-list/UserListV2'
+
+type FollowersUserListProps = {
+  getScrollParent: () => HTMLElement | null
+  onClose: () => void
+}
+
+export const FollowersUserList = ({
+  onClose,
+  getScrollParent
+}: FollowersUserListProps) => {
+  const userId = useSelector(followersUserListSelectors.getId)
```

Comment:
we could probably migrate this to use a useModal now

pass in the userId on open, and call the modal's hook here to get the id